plugins/src/main/java/com/google/firebase/gradle/plugins/FirebaseJavaLibraryPlugin.kt

```diff
@@ -120,12 +119,12 @@ class FirebaseJavaLibraryPlugin : BaseFirebaseLibraryPlugin() {
   private fun setupApiInformationAnalysis(project: Project) {
     val srcDirs =
       project.files(
-        project.convention
-          .getPlugin<JavaPluginConvention>()
+        project.extensions
+          .getByType(JavaPluginExtension::class.java)
```

Comment:
You can use the util method here

```suggestion
          .getByType<JavaPluginExtension>()
```

But you'll need to add the dsl import
```kt
import org.gradle.kotlin.dsl.getByType
```

Reply:
Done